includes/class-item-base.php

```diff
@@ -66,12 +66,12 @@ function __construct( $args = array() ) {
 
 				}
 
-				if ( method_exists( $class_name, 'make_new' ) ) {
+				if ( method_exists( $class_name, 'make_new_item' ) ) {
 					/*
 					 * If this class has a make_new() method.
-					 * A Class::make_new() method is needed when the class constructor has more than 1 parameter of $args.
```

Comment:
Somehow we ended up with both `make_new()` and `make_new_item()` in WPLib and looking at the uses in the library I am not 100% sure of the different between them.  It there is a difference between the two it would be good to flesh it out and document it.  If not, I would prefer to move all to 'make_new()` instead of `make_new_item()`.

The location above where you have made the change the class instance that is being created it either a model or a view, so "_make new **item**"_ would be wrong.